etc/p3-config/p3-config.yml

```diff
@@ -403,6 +405,19 @@ p3_image_ubuntu:
     os_distro: "ubuntu"
     os_version: 16.04
 
+# Latest Ubuntu image with CUDA
+p3_image_ubuntu_cuda:
+  name: "Ubuntu16-CUDA"
+  elements:
+    - "ubuntu"
+    - "nvidia-cuda"
+    - "vm"
+  env:
+    DIB_NVIDIA_CUDA_DELETE_REPO: "n"
```

Comment:
I guess this is to ensure updates get applied correctly?

Reply:
Correct - it's a publicly accessible repo, so we can leave it in place and users can refresh the packages if they wish.